features/githubActionRunner/file.include/etc/kernel/cmdline.d/99-no-lsm.cfg

```diff
@@ -1 +0,0 @@
-CMDLINE_LINUX="$(echo "$CMDLINE_LINUX" | tr ' ' '\n' | grep -v '^security=' | tr '\n' ' ' | sed -e 's/ $//')"
```

Comment:
please keep this file for now, until we get exclude features to work correctly

Reply:
true, forget to re-include it again. thanks.